shared-ts/disruptionTypes.zod.ts

```diff
@@ -775,5 +775,6 @@ export const disruptionSchema = disruptionInfoSchemaRefined.and(
             })
             .optional(),
         publishStatus: z.nativeEnum(PublishStatus).default(PublishStatus.draft),
+        template: z.boolean().optional(),
```

Comment:
It might be a good idea to set a default value of false:  ```template: z.boolean().optional().default(false)```

Reply:
done